world/client.cpp

```diff
@@ -538,6 +538,16 @@ bool Client::HandleNameApprovalPacket(const EQApplicationPacket *app)
 	uchar race = app->pBuffer[64];
 	uchar clas = app->pBuffer[68];
 
+	if (race < 0 || race > 255) {
+		LogInfo("Client::HandleNameApprovalPacket Race was less then zero or over 255");
```

Comment:
Drop `Client::HandleNameApprovalPacket` as it comes with logging for free now

Reply:
Wouldn’t Froglok/Drakkin be higher than 255? I’d just do `if (!IsPlayerRace(race))`.